CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 ### Added
+- `ignore_any_from_errors` option to suppress `no-any-expr` messages from other errors
```

Comment:
i wouldf suggest adding some more documentation to clarify what this means

Reply:
I'm gonna update the docs when it's merged.